openquake/sep/landslide/nowicki_jessee.py

```diff
@@ -0,0 +1,164 @@
+from typing import Union
+import numpy as np
+
+
+LANDCOVER_TABLE={
+    b"14": 0.91,
+    b"20": 0.88,
+    b"30": 0.78,
+    b"40": 0.68,
+    b"50": 0.30,
+    b"60": 1.77,
+    b"70": 1.71,
+    b"90": -1.26,
+    b"100": 1.50,
+    b"110": 0.68,
+    b"120": 1.13,
+    b"130": 0.79,
+    b"140": 1.03,
+    b"150": 0.54,
+    b"160": 2.34,
+    b"170": 1.19,
+    b"180": 1.19,
+    b"190": 0.30,
+    b"200": -0.06,
+    b"220": -0.18,
+    b"230": -1.08,
+    "14": 0.91,
```

Comment:
It would be good to check if we can find a way to avoid the duplication of coefficients for class codes represented as bytes or strings, because we're introducing more surface for errors to creep in.